### DISCUSSION TITLE
Request for Participation: We need developers!

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/ACE_Framework#32] [General]
Hey everyone, we're looking for 2 developers to join the core team on a volunteer basis.

1. Python dev with Flask and/or Django experience
2. Web designer (UI/Tailwind)

Some other considerations:

- Must have at least 10 hours per week (up to 8 weeks)
- Preferably Western Hemisphere (North or South America)
  - Potentially European time zone (we may have an Americas and European team)
  - This is due to timezone conflicts and ensuring the teams can work together synchronously

How to apply:

1. Post your experience, expertise, etc here in this thread.
2. Ping @lancecarlson in your post
3. Make sure to be detailed! Provide links to your work, etc. Short messages that don't give us much to go on are likely to be glanced over. 
4. Even if we don't select you for the core team, your expertise could be valuable in another team!

**Comment by samgriek** [comment]


**Reply by lancecarlson** [reply]
Hi @samgriek ! Can you reply with your discord username? Let's video chat!

**Comment by samgriek** [comment]
> Hi @samgriek ! Can you reply with your discord username? Let's video chat!

Sure!

**Comment by GWSzeto** [comment]
@lancecarlson I would like to apply for the UI/Tailwind position. I have worked at 2 early stage startups, [MedMe Health](https://www.medmehealth.com/) and [Dyve](https://dyve.xyz/). At both startups, I have worked on building the systems and UI from the ground up. For the past 2 months, I have been working as a freelance developer in the web3 space along with building independent projects.

My most recent work is a project called Tweet Trader. A demo of the project can be found [here](https://youtu.be/fMO4-bzIHMA) and the source code can be found [here](https://github.com/orgs/Tweet-Trader/repositories)

**Reply by lancecarlson** [reply]
@GWSzeto hi! Can you send me your discord username?

**Reply by GWSzeto** [reply]
sz#7160

**Comment by Nabeegh-Ahmed** [comment]
Hey @lancecarlson I would like to apply for both of the positions. 

Here is my experience: 
<img width="644" alt="image" src="https://github.com/daveshap/ACE_Framework/assets/29403249/5f916807-de7f-4226-a73b-275e0e3c0cf0">

I have primarily worked on engineering and development but more recently I have started to develop a strong foundation in AI research. I have a good understanding of the latest developments in AI (LLMs mostly). You can look at my Github profile to learn more about my technical skills and expertise. 

Thanks!
Nabeegh Ahmed

**Comment by Qriscuit** [comment]
Hey guys, my name is Hursh. I am a Game Developer and have been working with OOP design patterns like State Machines and in Server Authoritative Multiplayer Architectures. Recently I've been getting my hands dirty with Django, Langchain, Sqlite and Qdrant to create a reliable vector search because its going to be quite important to have very good vector search for an ACE. My friend is working on the front end with Vue as his preferred choice. 

I am definitely not right for the right guy to be the principle engineer here because of my inexperience. I still wanna help so this is out to the python dev who gets selected. Let me be your grunt work programmer. You can put in 5-10 hours in a week ill add in 15 for the week

(went to college for Game Design, graduated with primary expertise in Game Programming, just been 2 years out of college)

what do you get:
1. A decent programmer that can laydown the API level functions for the greater architecture to access.
2. A receptive wall to bounce ideas off of
3. Commitment; My friend and i have been working with the goal of creating an AGI or an ACE for months. Without formal software development training we are trying to learn as much as we can, our sources are videos, books and communities. This is also very fundamental learning for me and I'll give it the work ethic it requires.

Regardless of whether i get to help you guys or not, we will create a safe ACE for everyone to use

**Comment by agusavior** [comment]
Hi there! Auto-scaling, Cloud Engineer Scaling Expert here!
I can help by building the infrastructure of the entire project! contact me! I win by comission/equity/directy pay.
If you don't have the MVP yet ready, I also can help with that, I am trying to build AGI from my teens! 🚀💪

**Comment by ayabdi** [comment]
Hey peeps 👋

Im Abdul, Im a full-stack software dev. I've spent the past 3 years hacking away at a couple of venture-backed pre-seed startups (YC, Techstars).

I've been following David's work for a couple of months now and would love to be a part of any future teams forming (preferably in the European timezones). Happy and more than able to contribute across the stack.

To tell you a bit about what I've been working on, we're currently building on a sales assistant for cold-callers at my current workplace. An assistant that can vet leads, do prospecting, objection handling, etc. Also, here's a little project I worked on earlier in the year [link](https://github.com/ayabdi/memoria)

**Comment by dvnpremoutlook** [comment]
HI @lancecarlson 

I am unsure if this position is still available, I am currently doing my masters in AI at SJSU, I have 3+ years of experience in AI and I have 1 year of experience as researcher in AI at SJSU

please let me know

I have attached my resume to this post 

Thank you
[Prem Dommalapati Resume (1).pdf](https://github.com/daveshap/ACE_Framework/files/13304279/Prem.Dommalapati.Resume.1.pdf)

**Comment by yattishr** [comment]
Hey!
I am a self-taught dev with none of the fancy experience that most full-stack developers have. But, I am enthusiastic, willing to learn, can learn, and adept at finding & fixing bugs. I have experience with Python, NodeJs, React and Next. I am currently writing a book on large language models and generative AI. I'm self-employed, so I have time to spare in the week and weekends with whatever menial tasks there might be for me to jump in and get my hands dirrrrrty.

**Comment by Qriscuit** [comment]
Hi my name's Hursh I'm a game dev but i have some experience with Django, Generative AI abd Vector Databases

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by MichaelReaNZ** [comment]
I'm interested, I've already started making an ACE using Next.js & Convex. I think it's important to be able to view what the layers are thinking in a UI.

Is there a discord I can join to share what I've done?
